omisego-sdk/src/main/java/co/omisego/omisego/websocket/listener/TransactionConsumptionListener.kt

```diff
@@ -14,8 +14,9 @@ import co.omisego.omisego.websocket.event.TransactionConsumptionFinalizedEvent
 import co.omisego.omisego.websocket.strategy.FilterStrategy
 
 abstract class TransactionConsumptionListener(
-    final override val strategy: FilterStrategy = FilterStrategy.Event(allowedEvents)
+    transactionConsumption: TransactionConsumption
```

Comment:
Is the topic unique for a given `TransactionConsumption`?
If not can't we just use the constant in the declaration and make it a no-arg constructor? (would make more sense than associating a listener to a given `TransactionConsumption` when it can in fact receive events for another `TransactionConsumption` object

Reply:
Yeah, it should be unique because a new topic will be created when consume the transaction.